src/c-writer.cc

```diff
@@ -226,7 +226,7 @@ class CWriter {
   static const char* GetReferenceTypeName(const Type& type);
   static const char* GetReferenceNullValue(const Type& type);
 
-  enum class WriteExportsKind {
+  enum class WriteCStmtKind {
```

Comment:
I wonder if we can find a better name?   WriterPhase?

Reply:
CWriterPhase?

Reply:
`CWriterPhase` sounds good